libcudacxx/include/cuda/__ptx/instructions/generated/multimem_red.h

```diff
@@ -775,56 +775,56 @@ multimem_red(sem_t<_Sem> __sem, scope_t<_Scope> __scope, op_and_op_t, _B32* __ad
   {
     asm("multimem.red.relaxed.cta.global.and.b32 [%0], %1;"
         :
-        : "l"(__as_ptr_gmem(__addr)), "r"(__as_b32(__val))
```

Comment:
@ahendriksen there are a lot of additional changes that look like a refactoring. Does this look correct to you? If yes, we may still want to pull this out as an extra PR.

Reply:
Refactoring split off to a new PR: #3632